# Request
---

# Response
sensor_msgs/Image image
string species_name
bool isok